packages/mgt-components/src/components/mgt-file-list/mgt-file-list.ts

```diff
@@ -760,6 +759,21 @@ export class MgtFileList extends MgtTemplatedComponent {
     }
   }
 
+  /**
+   * Handles keyup presses events.
+   *
+   * @param event
+   */
+  private onFileListKeyUp(event: KeyboardEvent): void {
+    const fileList = this.renderRoot.querySelector('.file-list');
+
+    if (event.code === 'Tab') {
+      if (this._focusedItemIndex === -1) {
+        this._focusedItemIndex = fileList?.children.length;
```

Comment:
see this seems weird to me, shouldn't this just be setting _focusedItemIndex to 0?
As we know that this is only on the first child in this file list. The current code sets this to be the length of the files list which is actually outside of the bounds of the array.